parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -90,7 +87,7 @@ public void visit(ThriftType.MapType mapType) {
     //=========handle value
     currentFieldPath.push(mapValueField);
     currentName = "value";
-    currentRepetition = OPTIONAL;
+    currentRepetition = OPTIONAL; // TODO: why?
```

Comment:
Why are map value optional? If they get projected out, we replace them with null?